src/box/replication.cc

```diff
@@ -231,6 +300,12 @@ replication_init(int num_threads)
 	replicaset.healthy_count = 1;
 
 	applier_init();
+	replication_anon_gc_fiber =
```

Comment:
Shouldn't you also shutdown this fiber in `replication_free()`?

Reply:
Oh... Thanks! Shutdown it in `replication_shutdown`.